src/buildercore/cfngen.py

```diff
@@ -279,6 +292,15 @@ def write_template(stackname, contents):
     open(output_fname, 'w').write(contents)
     return output_fname
 
+def write_terraform_template(stackname, contents):
+    "optionally, store a terraform configuration file for the stack"
+    if contents:
```

Comment:
`if` allows other stacks not to be affected by what is going on here, making the Terraform files effectively optional